src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2726,14 +1796,9 @@ bool processOnChangeConditionForSubscription
   Restriction                   res;
   ContextElementResponseVector  rawCerV;
 
-  /* Include scopes for entitiesQuery() if q filter is not empty */
```

Comment:
Yes, this has been modified, the string filter is now passed via Scope

Fixed in some commit, March 22, I think ...